packages/beacon-node/src/api/impl/beacon/blocks/index.ts

```diff
@@ -304,9 +304,9 @@ export function getBeaconBlockApi({
         }
       } else if (
         chain.emitter.listenerCount(routes.events.EventType.dataColumnSidecar) &&
-        blockForImport.blockData.fork === ForkName.fulu
+        isForkPostFulu(blockForImport.blockData.fork)
       ) {
-        const {dataColumns} = blockForImport.blockData;
+        const {dataColumns} = blockForImport.blockData as BlockInputDataColumns;
```

Comment:
it's pretty strange that this breaks type inference, I was digging into it a while ago but couldn't figure it out so I guess this change is fine, at least we don't blow up when gloas comes around